### PR TITLE
Configure new CC test reporter

### PR DESCRIPTION
**Pull request by pbrisbin**
/cc @codeclimate/review

This was extracted from #122 where the DNS stubbing we're doing breaks the
integrated test reporter.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,3 +4,8 @@ source "https://rubygems.org"
 gemspec
 
 gem "pry"
+
+group :test do
+  gem "simplecov"
+  gem "codeclimate-test-reporter", "1.0.0.pre.rc2"
```
Comment (dblandin) [inline Gemfile]: Is there an important distinction between keeping these in the gemspec as development dependencies and moving them here to a test group?

Comment (pbrisbin) [inline Gemfile]: No idea. Doing it this way was easier copy/pasting from the instructions.

Comment (dblandin) [inline Gemfile]: Ok :)

**Comment by dblandin** [comment]
LGTM

**Pull request merged**